### PR TITLE
Feature/227 common metrics

### PR DESCRIPTION
**Pull request by crbrox**
Implements #227 

@fgalan

### CODE REVIEW

File: lib/models/metrics.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    'incomingTransactionsErrors',
+
+    'serviceTime',
+
+    'outgoingTransactions',
+    'outgoingTransactionsRequestSize',
+    'outgoingTransactionsResponseSize',
+    'outgoingTransactionsErrors',
+
+    'notifications',
```
Comment (fgalan) [inline lib/models/metrics.js]: The original scope at issue #227 didn't include component-specific metrics. However, at the end they has been included.

NTC (just informative)

File: config.js [hunk 2]
```diff
@@ -44,7 +44,8 @@ config.endpoint = {
     vrPath : '/m2m/vrules',
```
Comment (fgalan) [inline config.js]: Suggestion:

"Add: metrics REST API (#227)"

File: lib/models/metrics.js [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+    decorated.sum = total;
+    return decorated;
+}
+
+module.exports.GetMetrics = getMetrics;
+module.exports.DeleteMetrics = deleteMetrics;
+module.exports.IncMetrics = incMetrics;
+module.exports.GetDecorated = decoratedMetrics;
+
+module.exports.incomingTransactions = 'incomingTransactions';
```
Comment (crbrox) [inline lib/models/metrics.js]: Also not easy to simplify the use in the array ... we could en writing the same name 3 times ...

I'd left as it is just in this case

Comment (fgalan) [inline lib/models/metrics.js]: NTC

File: config.js (hunk 2 above)
Comment (fgalan) [inline config.js]: Fixed in e8b455fdd0a9eb7c4b0fd907df3c7d6bd28c3662

**Comment by fgalan** [comment]
LGTM

**Pull request merged**